pages/trust-admin/hospitals/index.js

```diff
@@ -9,11 +9,22 @@ import Heading from "../../../src/components/Heading";
 import ActionLink from "../../../src/components/ActionLink";
 import Text from "../../../src/components/Text";
 
-const TrustAdmin = ({ hospitals, hospitalError, trust, trustError }) => {
+const TrustAdmin = ({
+  hospitals,
+  hospitalError,
+  trust,
+  trustError,
+  hospitalVisitTotals,
+}) => {
   if (hospitalError || trustError) {
     return <Error err={hospitalError || trustError} />;
   }
 
+  const hospitalsWithVisitTotals = hospitals.map((hospital) => {
```

Comment:
What do you think about moving this to `getServerSideProps` so that we can test that `hospital.bookedVisits` is set correctly ?

Reply:
YES!